src/test/java/TestStorage.java

```diff
@@ -0,0 +1,65 @@
+import Collections.CustomLinkedList;
+import Collections.CustomList;
+import Collections.List;
+import Containers.StorageLoadResult;
+import Containers.StorageSaveResult;
+import Containers.StorageType;
+import Containers.StorageUnit;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TestStorage {
+
+    @Test
+    void testFileStorageNotLinked(){
+        CustomList myList = new CustomList();
+        testFileStorage(myList, false);
+    }
+
+    @Test
+    void testFileStorageLinked(){
+        CustomLinkedList myList = new CustomLinkedList();
+        testFileStorage(myList, true);
+    }
+
+    @Test
+    void testDatabaseStorageNotLinked(){
```

Comment:
test name should tell what is happening, and what is testing

I can't understand what is been tested here by name